django_test_timer/__init__.py

```diff
@@ -24,10 +24,7 @@ def addSuccess(self, test):
 
 # Django agnostic
 class TimedUnitTestRunner(unittest.TextTestRunner):
-
-    def __init__(self, *args, **kwargs):
-        kwargs['resultclass'] = TimeLoggingTestResult
-        return super(TimedUnitTestRunner, self).__init__(*args, **kwargs)
+    resultclass = TimeLoggingTestResult
```

Comment:
[The docs](https://docs.python.org/2.7/library/unittest.html) don't mention setting `resultclass` directly on the class (even though it may work) so I think best practice would be to keep it passed in `__init__` and default to `TimeLoggingTestResult` if none if passed.

```suggestion
    def __init__(self, *args, **kwargs):
        kwargs['resultclass'] = kwargs.get('resultclass', TimeLoggingTestResult)
        return super(TimedUnitTestRunner, self).__init__(*args, **kwargs)
```

Reply:
I've reinstated `__init__` - let me know if this solution works for you.